shared/reducers/chat2.tsx

```diff
@@ -54,32 +55,6 @@ const messageIDToOrdinal = (
   return null
 }
 
-const passToTeamBuildingReducer = (
-  draftState: Container.Draft<Types.State>,
-  action: TeamBuildingGen.Actions
-) => {
-  draftState.teamBuilding = teamBuildingReducer(
-    'chat2',
-    draftState.teamBuilding as Types.State['teamBuilding'],
-    action
-  )
-}
-
-const teamActions: Container.ActionHandler<Actions, Types.State> = {
```

Comment:
moved to a helper so its not copy/pasted / leaves a new action getting missed somewhere possibly